README.md

```diff
@@ -13,6 +13,16 @@ npm install --save redux-actions
 import { createAction, handleAction, handleActions } from 'redux-actions';
 ```
 
+### Note
```

Comment:
I think this still needs to be made smaller: https://github.com/acdlite/redux-actions/pull/91#discussion_r67833083

Either `####`, or just prefix the statement with a less intrusive **Note:**.

Another nit on English:

> We also support a UMD build